webpack/assets/javascripts/react_app/components/AuditsList/SearchLink.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class SearchLink extends React.Component {
+  render() {
+    const {
+      url,
+      title,
+      id,
+      textValue,
+    } = this.props;
+
+    const linkProps = {
+      href: url,
+      title,
+      'data-toggle': 'tooltip',
```

Comment:
@amirfefer,

I have tried to resolve this. In case of above search link and 'Host Details' link, it won't display any tooltip using EllipsisWithTooltip. In case of links this condition `el.scrollWidth > el.clientWidth` fails everytime i.e at 
https://github.com/theforeman/foreman/pull/5869/files/3428aeb929ef7c1adf734fc5aaa3666f9a62be59#diff-243fc7f0898dd0a8e95fe0a89f977c35R19 

Please let me know if I am missing anything here.